openpype/tools/settings/settings/base.py

```diff
@@ -325,7 +341,11 @@ def _apply_values_from_project_action(self, menu, actions_mapping):
 
             action = QtWidgets.QAction(project_name)
             submenu.addAction(action)
-            actions_mapping[action] = lambda: self._apply_values_from_project(
+            # Use custom callback object instead of lambda
+            #   - project_name value is changed each value so all actions will
+            #       use the same source project
+            actions_mapping[action] = _Callback(
+                self._apply_values_from_project,
                 project_name
             )
```

Comment:
Shouldn't this just use `functools.partial(self._apply_values_from_project, project_name)` instead of its own Callback wrapper class?

Reply:
Yep... I forgot about `functools`.

Reply:
Changed in https://github.com/pypeclub/OpenPype/pull/2825